code/game/objects/items/devices/radio/beacon.dm

```diff
@@ -85,4 +92,5 @@
 		new /obj/machinery/telepad_cargo(user.loc)
 		playsound(src, 'sound/effects/pop.ogg', 100, 1, 1)
 		qdel(src)
-	return
\ No newline at end of file
+	return
+	
```

Comment:
This file appears to end as git likes. But then has a tab. Which git doesn't like.